assets/scss/components/_form.scss

```diff
@@ -21,3 +25,230 @@
+    display: flex;
+    padding: 16px;
+    align-items: center;
+    gap: 8px;
+    align-self: stretch;
+    border-radius: 4px;
+    border: 1px solid $color-neutrals-75;
+    background: $color-white;
+    color: $color-neutrals-500;
+    font-family: $font-sans-serif;
+    font-size: 16px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 1.5;
+    -webkit-appearance: none;
+    -moz-appearance: none;
+    appearance: none;
+    transition: all 0.2s ease-in-out;
+    background: url($dropdown-icon) no-repeat;
+    background-position: calc(100% - 16px) center;
+    background-size: 24px;
+
+    &:hover {
+        border: 1px solid $color-primary-300;
+        transition: all 0.2s ease-in-out;
+    }
+
+    &:focus {
+        box-shadow: 0px 0px 0px 4px rgba(90, 100, 193, 0.24);
+        transition: all 0.2s ease-in-out;
+    }
+}
+
+label {
+    color: $color-black;
+    font-family: $font-sans-serif;
+    font-size: 16px;
+    font-style: normal;
+    font-weight: 600;
+    line-height: 1.5;
+    margin-bottom: 8px;
+
+    small {
+        color: $color-red-300;
+        font-weight: 400;
+    }
+}
+
+.js-form-field-error {
+    label,
+    .js-inline-message {
+        color: $color-red-300;
+    }
+}
+
+.js-inline-message {
+    font-family: $font-sans-serif;
+    font-size: 16px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 1.5;
+}
+
+.js-form-field-radio {
+    .js-input[type='radio'] {
+        display: none;
+    }
+
+    label:not(:first-child) {
+        padding-left: 32px;
+        position: relative;
+        cursor: pointer;
+
+        &::before {
+            content: '';
+            position: absolute;
+            left: 0;
+            top: -2px;
+            width: 24px;
+            height: 24px;
+            border-radius: 50%;
+            background: $color-white;
+            border: 1px solid $color-neutrals-75;
+            transition: all 0.2s ease-in-out;
+        }
+
+        &:hover {
+            &::before {
+                border: 1px solid $color-primary-300;
+                transition: all 0.2s ease-in-out;
+            }
+        }
+
+        &:focus {
+            &::before {
+                box-shadow: 0px 0px 0px 4px rgba(90, 100, 193, 0.24);
+                transition: all 0.2s ease-in-out;
+            }
+        }
+    }
+
+    .js-input[type='radio'] {
+        &:checked {
+            + label {
+                &::before {
+                    background-color: $color-white;
+                    border: 6px solid $color-primary-300;
+                }
+
+                &:hover {
+                    &::before {
+                        border-color: $color-primary-400;
+                    }
+                }
+
+                &:focus {
+                    &::before {
+                        box-shadow: 0px 0px 0px 4px rgba(90, 100, 193, 0.24);
+                    }
+                }
+            }
+        }
+    }
+}
+
+.js-form-field-checkbox {
```

Comment:
Classes with `.js-(...)` seem more suitable for linking js actions than styling

Reply:
There are instances where the styling of an error message in the form depends on the element with class with '.js-form-field-error' which is the class that is added to that element via JavaScript when an error occurs. Without this class if we apply error styling to that element, it will be shown even if the message is not an error message. I think we are good to use these classes as well and we shouldn't be constrained. Otherwise we will spend lots of time adding other classes and modifying most of the structure again, which is not something we should focus on this project. We should just focus on styling it to make it match the designs. But if you want maybe we should discuss with either Pawel or Marcin (if he is available) to see how to approach this

Reply:
It makes sense. I will see which ones can be edited and I will add different classnames to seperate logic from js class names. Thanks Ola